docs/extending.md

```diff
@@ -366,6 +366,7 @@ and reduce the bloat in your own codebase.
 [fake-object]: https://c2.com/cgi/wiki?FakeObject "Fake Object"
 [implementation-documentation]: javadoc/latest/org/robolectric/annotation/Implementation.html
 [implements-documentation]: javadoc/latest/org/robolectric/annotation/Implements.html
+<!-- markdownlint-disable-next-line MD053 -->
```

Comment:
This link reference is actually used, but for some reason, not detected by the linter.

https://github.com/DavidAnson/markdownlint/blob/v0.36.0/doc/md053.md